src/crs_linter/linter.py

```diff
@@ -104,8 +109,12 @@ def __init__(self, data, filename=None, txvars={}):
         self.noveract = []  # list of rules without ver action or incorrect ver
         self.nocaptact = []  # list of rules which uses TX.N without previous 'capture'
 
-        self.re_tx_var = re.compile(r"%\{\}")
-        self.filename = filename
+    def is_error(self):
+        """ Returns True if any error is found """
+        return len(self.caseerror) > 0 or len(self.orderacts) > 0 or len(self.auditlogparts) > 0 or len(
```

Comment:
error_vars=[self.caseerror, self.orderacts...]
return `any([len(var) > 0 for var in error_vars])`